src/utilities/algolia.js

```diff
@@ -196,8 +197,7 @@ const stateSettings = {
  * (see https://www.algolia.com/doc/guides/building-search-ui/going-further/backend-search/how-to/highlighting-snippeting/)
  */
 const pageSettings = {
-  attributeForDistinct: 'section',
-  distinct: true,
+  attributeForDistinct: 'contentful_id',
```

Comment:
@kevee I think this may lead to record dupes for the same page (https://www.algolia.com/doc/guides/sending-and-managing-data/prepare-your-data/how-to/indexing-long-documents/#after) as `contentful_id` would be the same across different chunks of a single page. If I got the docs right which is not sure at all.

Reply:
Right, I tested this and if you don't set `distinct` to true, then it should just say "use contentful_id when doing a distinct query" but the blog records all go into the index.

Reply:
@superbiche - can you check the algolia test_ indexes and see if the blogs look right?

Reply:
@kevee looks good !